pco/src/bin_optimization.rs

```diff
@@ -42,6 +47,11 @@ fn choose_optimized_partitioning<L: Latent>(
   let total_count = c;
   let lowers = bins.iter().map(|bin| bin.lower).collect::<Vec<_>>();
   let uppers = bins.iter().map(|bin| bin.upper).collect::<Vec<_>>();
+  // On Windows, log2() is very slow, so we use log(2.0) instead, which is
+  // about 10x faster. On other platforms, we stick with log2(). See #223.
+  #[cfg(target_os = "windows")]
```

Comment:
This is only run once per compression, so let's not complicate the code in this part.